snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -44,7 +44,7 @@
     import urllib.request
     import sys
 
-    check_url = "http://start.ubuntu.com/connectivity-check.html"
+    check_url = "http://connectivity-check.ubuntu.com/"
```

Comment:
This is fine, I pinged the web team for some backwards compatibility for the time being too